packages/arb-token-bridge-ui/src/components/TransactionsTable/TransactionsTable.tsx

```diff
@@ -193,6 +193,9 @@ const TableRow = ({ tx }: { tx: MergedTransaction }): JSX.Element => {
     if (nodeBlockDeadline === 'NODE_NOT_CREATED') {
       return l2Network.chainID === 42161 ? '~ 1 week' : '~1 day'
     }
+    if (nodeBlockDeadline === 'EXECUTE_CALL_EXCEPTION') {
```

Comment:
weird that it can have this value, we should add it to `NodeBlockDeadlineStatus`
https://github.com/OffchainLabs/arb-token-bridge/blob/9742dc6faff37a45fc1eae75bbb1e06a085cc52e/packages/token-bridge-sdk/src/hooks/arbTokenBridge.types.ts#L34

Reply:
You are on outdated commit
https://github.com/OffchainLabs/arb-token-bridge/blob/11118fd3e04e331c7aaed1ceb4123c8846b1fe53/packages/token-bridge-sdk/src/hooks/arbTokenBridge.types.ts#L37